trustregistry/main.py

```diff
@@ -19,10 +24,59 @@
 ROOT_PATH = os.getenv("ROOT_PATH", "")
 
 
+def check_migrations(
+    engine: Engine, alembic_cfg: Config  # pylint: disable=redefined-outer-name
```

Comment:
This `redefined-outer-name` is because we name the param `engine`, and also import `from trustregistry.database import engine`. Better would be to rename one of those two.